cfme/tests/intelligence/reports/test_reports_schedules.py

```diff
@@ -92,6 +93,21 @@ def schedule(schedule_data, appliance):
     schedule.delete_if_exists()
 
 
+@pytest.fixture
+def user(appliance, request):
+    users, user_data = _users(
+        request,
+        appliance,
+        name="Sherlock Holmes",
+        email="shholmes@redhat.com",
```

Comment:
Can't wait for the day this username exists on redhat email servers, and this test executes in an environment that can reach them :)